internal/build/phase_config_provider.go

```diff
@@ -244,6 +244,12 @@ func WithRoot() PhaseConfigProviderOperation {
 	}
 }
 
+func WithoutPrivilege() PhaseConfigProviderOperation {
+	return func(provider *PhaseConfigProvider) {
+		provider.ctrConf.User = "1000:1001"
```

Comment:
Rather than hard-code the uid/gid we can get this information from the builder image

Reply:
Update: maybe we can just get rid of this operation. `CNB_USER_ID` should still be set in the builder env, and the lifecycle will drop privileges to that user before running the build command.

Reply:
Or we could use `l.opts.Builder.UID()` etc

Reply:
done